scripts/core_tests_list.json

```diff
@@ -33,7 +33,8 @@
     "test_icommands_file_operations.Test_ICommands_File_Operations_3",
     "test_icommands_file_operations.Test_ICommands_File_Operations_4",
     "test_icommands_file_operations.Test_ICommands_File_Operations_5",
-    "test_icp",
+    "test_icp.Test_Icp",
+    "test_icp.test_icp_overwrite_with_target_resource",
```

Comment:
is this correct?   these two lines look different... capitalization, etc...

Reply:
Whups, I did rename the class. But the first one is correct. Second one should be `test_icp.test_overwrites`.

Reply:
Done